### PR TITLE
Resources to variables & updated box version

### LINKED ISSUE #49: Add resources to variables
## Feature description
Set all resources with the help of variables, so that users can set their own numbers if they wish.

## Why is it needed?
More flexibility

## Suggestion(s)/solution(s) [Optional]
Add variables to `variables.tf`, and add a resource stanza to the hcl

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### LINKED ISSUE #51: Updgrade to 0.7.1 of box
## What is the issue?
Upgrade and test v0.7.0 of the hashistack box. 
https://app.vagrantup.com/fredrikhgrelland/boxes/hashistack/versions/0.7.0

## Suggestion(s)/solution(s) [Optional]
Change `Vagrantfile.default` versioning to fetch newest box between 0.7 and 0.8

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by claesgill**
## Closes/fixes/resolves issue(s)?
Closes #49 
Closes #51 
50% fixed issue #15 

## What was added/changed/fixed?
- Resources to variables
- Updated readme
- Updated box version
- Removed duplicate `dev` method in Makefile
- Role error fixed only when `use_vault_provider=false`

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
 ### Added
 - Added switch for canary deplyment #44
+- Added resource variables #51
 
 ### Changed
 - Removed box example [no issue]
 - Updated README with intentions section #46
 - Proper rendering of credentials #45
+- Updated box version #49
```
Comment (pdmthorsrud) [inline CHANGELOG.md:13]: I think I'd like to include this. I imagine somebody going back to look for when something was removed for example.
```suggestion
- Updated box version #49
- Removed `dev` target from makefile
```

Comment (claesgill) [reply CHANGELOG.md:13]: Fixing manually

**Review by pdmthorsrud — DISMISSED** [review]

**Review by zhenik — COMMENTED** [review]
In `/dev/vagrant/bootstrap/nomad/post/022-update-nomad-write-default-policy.yml`

You need update `path` 
```
cmd: vault kv get -field=secret-id secret/nomad-bootstrap-token
```
to 

```
cmd: vault kv get -field=secret-id secret/example/nomad-bootstrap-token
```

You can check the proper path in the Vault UI
Screenshot 2020-11-06 at 12 36 08

**Comment by zhenik** [comment]
Also, @claesgill check line 9 here /dev/ansible/04_test_persistence.yml

**Review by pdmthorsrud — DISMISSED** [review]

**Review by pdmthorsrud — APPROVED** [review]

**Review by zhenik — APPROVED** [review]

**Pull request merged**